src/NRedisStack/Json/IJsonCommands.cs

```diff
@@ -0,0 +1,232 @@
+﻿using StackExchange.Redis;
+
+namespace NRedisStack;
+
+public interface IJsonCommands
+{
+    /// <summary>
+    /// Appends the provided items to the array at the provided path.
+    /// </summary>
+    /// <param name="key">The key to append to</param>
+    /// <param name="path">The path to append to</param>
+    /// <param name="values">the values to append</param>
+    /// <returns>The new array sizes for the appended paths</returns>
+    /// <remarks><seealso href="https://redis.io/commands/json.arrappend"/></remarks>
+    public long?[] ArrayAppend(RedisKey key, string? path, params object[] values);
```

Comment:
The command name should be exactly as it is called in the documentation to avoid confusion.
Should be callded "ArrAppend".